paper-ieee-trans.tex

```diff
@@ -399,7 +400,7 @@ \section{New trends in software reliability}
 \subsection{Software Quality Procedures}
 \label{subsec:sqa}
 
-The software quality policies were initially defined in the first deliverable document of the
+The initial software quality policies were defined in the first deliverable document of the
```

Comment:
i prefer the master version

Reply:
agree with Pablo
The software quality policies were initially defined

Reply:
ok

Reply:
ok